infra/charts/feast/charts/feast-core/values.yaml

```diff
@@ -4,12 +4,12 @@
 postgresql:
   # enabled specifies whether Postgresql should be installed as part of Feast Core.
   #
-  # Feast Core requires a database to store data such as the created FeatureSets 
+  # Feast Core requires a database to store data such as the created FeatureSets
   # and job statuses. If enabled, the database and service port specified below
-  # will override "spring.datasource.url" value in application.yaml. The 
+  # will override "spring.datasource.url" value in application.yaml. The
   # username and password will also be set as environment variables that will
   # override "spring.datasource.username/password" in application.yaml.
-  enabled: true
+  enabled: false
```

Comment:
why disabled here? we need deploy new postgres db. Am I right?

Reply:
right these are the helm configs, I was just rendering them to copy to the jsonnet